data/south jobs.txt

```diff
@@ -0,0 +1,201 @@
+		random < 15
+		"combat rating" > 100
+	source
+		attributes "south" "rim"
+	destination
+		distance 2 3
+		attributes "south"
+	npc
+		government "Militia"
+		personality heroic
+		fleet "Small Militia"
+	npc
+		government "Militia"
+		personality heroic staying
+		system destination
+		fleet "Small Militia"
+	npc evade
+		government "Pirate"
+		personality heroic staying target harvests plunders
+		system destination
+		fleet "Small Southern Pirates" 3
+	on offer
+		conversation "pirate occupation"
+	on complete
+		payment 100000
+		dialog `The government of <planet> thanks you for defeating the pirates and pays you <payment>.`
+
+
+mission "Pirate Occupation [1]"
+	name `Pirates occupying <system>`
+	minor
+	repeat
+	description `A gang of pirates have been occupying <system>, and the local government has asked for help. Defeat the pirates in the system and land on <planet> when done. Payment is <payment>.`
+	to offer
+		random < 10
+		"combat rating" > 200
+	source
+		attributes "south" "rim"
+	destination
+		distance 2 3
+		attributes "south"
+	npc
+		government "Militia"
+		personality heroic
+		fleet "Small Militia" 2
+	npc
+		government "Militia"
+		personality heroic staying
+		system destination
+		fleet "Small Militia"
+	npc evade
+		government "Pirate"
+		personality heroic staying target harvests plunders
+		system destination
+		fleet "Small Southern Pirates" 2
+		fleet "Large Southern Pirates"
+	on offer
+		conversation "pirate occupation"
+	on complete
+		payment 150000
+		dialog `The government of <planet> thanks you for defeating the pirates and pays you <payment>.`
+
+
+mission "Pirate Occupation [2]"
+	name `Pirates occupying <system>`
+	minor
+	repeat
+	description `A large gang of pirates have been occupying <system>, and the local government has asked for help. Defeat the pirates in the system and land on <planet> when done. Payment is <payment>.`
+	to offer
+		random < 5
+		"combat rating" > 300
+	source
+		attributes "south" "rim"
+	destination
+		distance 2 3
+		attributes "south"
+	npc
+		government "Militia"
+		personality heroic
+		fleet "Large Militia"
+		fleet "Small Militia"
+	npc
+		government "Militia"
+		personality heroic staying
+		system destination
+		fleet "Small Militia"
+	npc evade
+		government "Pirate"
+		personality heroic staying target harvests plunders
+		system destination
+		fleet "Large Southern Pirates" 3
+	on offer
+		conversation "pirate occupation"
+	on complete
+		payment 200000
+		dialog `The government of <planet> thanks you for defeating the pirates and pays you <payment>.`
+
+conversation "pirate occupation"
+	`Suddenly you hear raised voices and shouting outside as pilots run for their ships: "<destination> is being occupied by a pirate fleet! They are requesting help from any combat-worthy ship in the region to join the defenses!" The authorities of <planet> will probably pay you quite well if you assist them, but this could also be an easy way to get yourself killed.`
+		choice
```

Comment:
I think this is indented one too far. Running the parse test throws this back:
```
Expected goto or endpoint in conversation, found this:
file "/home/tehhowch/projects/endless-sky/data/south jobs.txt"
  conversation "pirate occupation"
    `Suddenly you hear raised voices and shouting outside as pilots run for their ships: "<destination> is being occupied by a pirate fleet! They are requesting help from any combat-worthy ship in the region to join the defenses!" The authorities of <planet> will probably pay you quite well if you assist them, but this could also be an easy way to get yourself killed.`
      choice

Expected goto or endpoint in conversation, found this:
file "/home/tehhowch/projects/endless-sky/data/south jobs.txt"
  conversation "pirate occupation"
    `Suddenly you hear raised voices and shouting outside as pilots run for their ships: "<destination> is being occupied by a pirate fleet! They are requesting help from any combat-worthy ship in the region to join the defenses!" The authorities of <planet> will probably pay you quite well if you assist them, but this could also be an easy way to get yourself killed.`
      "	(Stay here until the fight is over.)"

Expected goto or endpoint in conversation, found this:
file "/home/tehhowch/projects/endless-sky/data/south jobs.txt"
  conversation "pirate occupation"
    `Suddenly you hear raised voices and shouting outside as pilots run for their ships: "<destination> is being occupied by a pirate fleet! They are requesting help from any combat-worthy ship in the region to join the defenses!" The authorities of <planet> will probably pay you quite well if you assist them, but this could also be an easy way to get yourself killed.`
      "	(Join the defense fleet.)"
```

Reply:
Probably the `launch` is too far indented as well. Parse test doesn't pick up on over-indentation at the moment.

Reply:
Yep, both `choice` and `launch` are overindented.

Reply:
Were*